zk_shell/shell.py

```diff
@@ -1085,6 +1086,14 @@ def do_get(self, params):
             except:
                 pass
 
+        # maybe it's lz4 compressed?
+        if value is not None:
+            try:
+                value = lz4.frame.decompress(value)
+            except BaseException as err:
+                print(err)
```

Comment:
this should either go to stderr or be swallowed

Reply:
Oops, that was from local testing, fixed now.